grafana-plugin/src/models/schedule/schedule.ts

```diff
@@ -530,10 +533,13 @@ export class ScheduleStore extends BaseStore {
   }
 
   @action.bound
-  async refreshEvents(scheduleId: string) {
+  async refreshEvents(scheduleId: string, scheduleView?: ScheduleView) {
```

Comment:
Can we set proper value of scheduleView in store instead of introducing this param?